src/Avalonia.Base/Animation/Animators/GradientBrushAnimator.cs

```diff
@@ -72,6 +73,25 @@ public override IDisposable BindAnimation(Animatable control, IObservable<IGradi
             return control.Bind((AvaloniaProperty<IBrush?>)Property, instance, BindingPriority.Animation);
         }
 
+        private static ImmutableTransform? InterpolateTransform(double progress,
+            ITransform? oldTransform, ITransform? newTransform)
+        {
+            if (oldTransform is TransformOperations oldTransformOperations
+                && newTransform is TransformOperations newTransformOperations)
+            {
+
+                return new ImmutableTransform(TransformOperations
+                    .Interpolate(oldTransformOperations, newTransformOperations, progress).Value);
+            }
+
+            if (oldTransform is { })
```

Comment:
This is a nit so feel free to ignore.

However, I wonder why are you all still using this syntax? It's widely not liked and is very unintuitive. Its usage only made some sense for a small period of time between language versions. With the addition of the "not" keyword (C# 9.0) it also really shouldn't be done anymore and most projects don't.

Besides, I think `!(oldTransform is null)` was always the most accepted syntax or more rarely `!object.ReferenceEquals(object, null)`. These are more readable alternatives.

Reply:
I guess it's just a habit. I would prefer "is not null" here, yes.